### DISCUSSION TITLE
Disabling the Recommended section is preventing to open apps on Start menu

### DISCUSSION THREAD

**Discussion by hlebushekhub** [valinet/ExplorerPatcher#1106] [Q&A]
As the title of the question says, when I try to hide recommendations on Start menu with File Explorer, I couldn't open any pin apps nor All apps tab. I can click shortcuts with right mouse somehow, but left click giving me nothing.

I'm on version 22612.1344.53.1 and Win 11 22H2

**Comment by valinet** [comment]
> 22612.1344.53.1

This EP version does not exist

> Win 11 22H2

Which build exactly...?

I cannot reproduce on OS build 22621.1413.
